config.json

```diff
@@ -96,5 +96,7 @@
     "hostname": "localhost",
     "signalLossTimeoutSeconds": 60,
     "csvExportOpenStreetMapsUrl": true
-  }
+  },
+  "HTTP_REQUEST_LISTEN_TO_YOLO_RETRY_DELAY_MS":30,
+  "HTTP_REQUEST_LISTEN_TO_YOLO_MAX_RETRIES":180
```

Comment:
Do you think we can put this under a YOLO or Darknet group and shorten the name a bit?

Also we should update the documentation to make sure all config options a properly described.

Reply:
E.g. we have the Darknet ports further down in the config. Maybe we can combine those in a new group?

Reply:
I put them under NEURAL_NETWORK_PARAMS as the labels were all related to YOLO.

Reply:
I mean, we could create a new group if you think that would be better?

Reply:
I guess the more I sit here and think about this, if we do the monitor below, then it would probably also need a set of parameters for how long, etc.  Maybe there should be a new group for time parameters or such.  Either way, easy to fix, so let me know the direction you want to go.

Reply:
Updating the Markdown files is enough for now. We can worry about docusaurus as we prepare for v4.0.0 release.

And I don't think we should introduce more "timeout" parameters for now. I think people will not care, and indeed be very happy that this is fixed!

Reply:
I'm going to move this out to a generic time group that can be used for any services that may need that.

Reply:
Documents have been updated and I moved the code to `MONITOR_SETTINGS` so they have their own section in the docs as well as in the config file.